packages/cactus-common/package.json

```diff
@@ -51,6 +51,7 @@
   },
   "dependencies": {
     "json-stable-stringify": "1.0.1",
+    "jsonwebtoken": "8.5.1",
```

Comment:
@outSH Could you please swap this out with something that also works in the browser? (webpack builds)